src/lib/components/compositions/input-stacked/stacked-label.svelte

```diff
@@ -27,7 +27,7 @@
   }
 </script>
 
-<div class="tw_input-spacing">
+<div class="tw_input-spacing tw_flex tw_flex-wrap">
```

Comment:
Fixing a bug when the password component is used with stacked labels, rather than floating.